aineko/cli/create_pipeline.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Utils to create a aineko pipeline from existing templates."""
+
+import os
+
+from cookiecutter.main import cookiecutter  # type: ignore
```

Comment:
```suggestion
from cookiecutter.main import cookiecutter
```
Please replace line 523 below with
`module = ["confluent_kafka", "confluent_kafka.*", "schema", "cookiecutter", "cookiecutter.*"]`
https://github.com/aineko-dev/aineko/blob/d472c3dee69ae26a4bafe62e6ead4ace73c13551/pyproject.toml#L522-L524